accounts/migrations/0015_auto_20171129_1537.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-11-29 15:37
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0014_oldusername'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='oldusername',
+            name='username',
+            field=models.CharField(db_index=True, max_length=255),
```

Comment:
because we check usernames with `__iexact` we need a case-insensitive index here: 
```
    "auth_user_username_upper_key" UNIQUE, btree (upper(username::text))
    "auth_user_username_gist_trgm_idx" gist (upper(username::text) gist_trgm_ops)
```
I can't remember why we have both a gist and a btree index, we should look into that and remove the one that we don't need.